crates/graphforge-bindings-py/tests/portable_v2_parity.py

```
@@ -0,0 +1,79 @@
+            assert "progress callback failed" in str(error)
+        assert events, "progress callback must run at least once"
+
+        try:
+            forge.export_portable_v2(
+                output_path=str(root / "not-callable.gfpb"),
+                representation="bundle",
+                profile="complete",
+                progress=object(),
+            )
+            raise AssertionError("expected non-callable progress to fail closed")
+        except Exception as error:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)